### PR TITLE
Added tutorial screens for Memory Match game

### LINKED ISSUE #1187: Add tutorial screen for mini game 2
## Description
As a developer,
I need to add tutorial screens for mini game 2 - Memory Match,
so that the user could be given proper instructions before the mini game is started.

## Mocks
The screens should have text views with proper instructions about the game.

### Update
- [ ]  initial xml file for tutorial screen

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
5 hours

### PR DESCRIPTION
**Pull request by Rimjhim28**
### Description
I have added xml layout for the tutorial of the mini game.
It contains three text views, viz. , txt_memory_tutorial_1, txt_memory_tutorial_2, txt_memory_tutorial_3 to give proper instructions to the user before starting the game.
I have used tag to have a proper background of mini game, that adapts properly to the screen, when the textviews are shown.
The logic I am planning to use here is change the value android:alpha to change the visibility of the design elements according to the tutorials that I will implement during coding.
Fixes #1187 

### Type of Change:

- User Interface

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?

The current view of the xml file is:

memory_tutorial

After modifications the screen would show three different tutorial texts on clicking on the screen:

35895361_1659479790836992_2515928886366175232_n

35829011_1659479844170320_1193009711440461824_n

35812500_1659479800836991_3332348280381112320_n

### Checklist:
**Delete irrelevant options.**

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: PowerUp/app/src/main/res/layout-land/activity_memory_match_game.xml [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    <ImageView
+        android:id="@+id/btn_start"
+        android:layout_width="@dimen/start_width_memory"
+        android:layout_height="@dimen/start_height_memory"
+        android:layout_marginBottom="@dimen/margin_large"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        android:src="@drawable/start_button" />
+
+    </android.support.constraint.ConstraintLayout>
```
Comment (Rimjhim28) [inline PowerUp/app/src/main/res/layout-land/activity_memory_match_game.xml]: This file is not related to this PR.
Changes appear here because of previous unmerged changes.

File: PowerUp/app/src/main/res/drawable/btn_bg.xml [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
     <stroke
         android:width="5dp"
-        android:color="@color/powerup_button_border" />
+        android:color="#ffffff" />
     <padding
         android:bottom="@dimen/btn_padding"
         android:left="@dimen/btn_padding"
```
Comment (thisum) [inline PowerUp/app/src/main/res/drawable/btn_bg.xml]: already merged. Pls remove

File: PowerUp/app/src/main/res/values/dimens.xml [hunk 3]
```diff
@@ -41,10 +41,5 @@
     <dimen name="memory_tile_size">120dp</dimen>
     <dimen name="memory_label_width">401dp</dimen>
     <dimen name="btn_margin_bottom">24dp</dimen>
-    <dimen name="tutorial_text_size">22sp</dimen>
-    <dimen name="tutorial_1_width">150dp</dimen>
-    <dimen name="tutorial_2_width">255dp</dimen>
-    <dimen name="tutorial_3_width">265dp</dimen>
-    <dimen name="tutorial_3_margin">32dp</dimen>
 </resources>
```
Comment (thisum) [inline PowerUp/app/src/main/res/values/dimens.xml]: You have used these 5 fields in `activity_kill_the_virus_tutorial.xml` file. It will give errors if you remove them here.

File: PowerUp/app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -108,7 +108,4 @@
     <string name="memory_label">Does the tile on the left match the tile that appeared one tile before it?</string>
     <string name="no">NO</string>
     <string name="yes">YES</string>
-    <string name="tutorial_1">Shoot the viruses with the syringe!</string>
-    <string name="tutorial_2">The color of the syringe should match with the color of the virus otherwise you loose!</string>
-    <string name="tutorial_3">Try to kill as many viruses as possible before the time runs out!</string>
 </resources>
```
Comment (thisum) [inline PowerUp/app/src/main/res/values/strings.xml]: You have used these 3 fields in activity_kill_the_virus_tutorial.xml file. It will give errors if you remove them here.
